src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -44,12 +44,11 @@ const ProfilePicture = ({
 let styles = StyleSheet.create({
   addCircle: {position: 'absolute', top: 25, left: 25},
   profileImage: {
-    width: 97,
-    height: 97,
+    width: dimen.normalizeDimen(92),
+    height: dimen.normalizeDimen(92),
     borderRadius: 100,
-    marginLeft: 1.4,
-    marginTop: 2,
-    marginBottom: 2
+    marginLeft: dimen.normalizeDimen(2.5),
+    marginTop: dimen.normalizeDimen(2.75)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The width and height of the `profileImage` style have been reduced, and the `marginLeft` and `marginTop` properties have been increased. These changes will result in a smaller profile image with more margin on the left and top. Make sure these changes align with your design specifications and don't cause any unexpected layout shifts.

```diff
-    width: 97,
-    height: 97,
-    marginLeft: 1.4,
-    marginTop: 2,
+    width: dimen.normalizeDimen(92),
+    height: dimen.normalizeDimen(92),
+    marginLeft: dimen.normalizeDimen(2.5),
+    marginTop: dimen.normalizeDimen(2.75)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->